infra/ursa/init-letsencrypt.sh

```diff
@@ -1,24 +1,21 @@
 #!/bin/bash
 
+if [ -z "$DOMAINS" ]; then
+    echo "Error: No domains provided."
+    echo "Please set the environment variable DOMAINS=(node.ursa.earth www.node.ursa.earth)"
+    exit 1
+fi
+
 if ! [ -x "$(command -v docker-compose)" ]; then
   echo 'Error: docker-compose is not installed.' >&2
   exit 1
 fi
 
-domains=(ursa.earth www.ursa.earth)
 rsa_key_size=4096
 data_path="./data/certbot"
 email="major@ursa.earth" # Adding a valid address is strongly recommended
 staging=1 # Set to 1 if you're testing your setup to avoid hitting request limits
 
-if [ -d "$data_path" ]; then
```

Comment:
why are we deleting this section?

Reply:
testing added it back with a option to skip via first arg